docs/source/conf.py

```diff
@@ -191,33 +191,49 @@ def __name__(self):
 # Options for HTML output
 # -----------------------
 
+# Use enthought-sphinx-theme if available
+try:
+    import enthought_sphinx_theme
+
+    html_theme_path = [enthought_sphinx_theme.theme_path]
+    html_theme = 'enthought'
+except ImportError as exc:
+    import warnings
+    msg = '''Can't find Enthought Sphinx Theme, using default.
+                Exception was: {}
+                Enthought Sphinx Theme can be downloaded from
+                https://github.com/enthought/enthought-sphinx-theme'''
+    warnings.warn(RuntimeWarning(msg.format(exc)))
+
+    # Use old defaults if enthought-sphinx-theme not available
+
+    # The name of an image file (within the static path) to place at the top
+    # of the sidebar.
+    html_logo = "e-logo-rev.png"
+
+    # The name of an image file (within the static path) to use as favicon of
+    # the docs.  This file should be a Windows icon file (.ico) being 16x16
+    # or 32x32 pixels large.
+    html_favicon = "et.ico"
```

Comment:
I'm getting a warning: `WARNING: favicon file 'et.ico' does not exist` here. Are we setting the path to the static files correctly?

Reply:
Okay, this isn't new in this PR; it's an issue on the master branch, too.

Reply:
Just out of interest, when does the warning come up? I couldn't see it when I was building the html docs just now, which is a bit strange.